samples/video/capture/sample.yaml

```diff
@@ -6,3 +6,8 @@ tests:
     tags: video
     platform_allow: mimxrt1064_evk
```

Comment:
That is a list of allowed boards so you can directly append mm_swiftio here.

Reply:
Thanks so much, fixed